common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -130,5 +130,10 @@
                 </a>
             </li>
         </ul>
+
+        <div class="new-header-feedback">
+            @fragments.inlineSvg("information-circle", "icon", List("main-menu__icon", "main-menu__icon--information"))
```

Comment:
It would make centering the icons a lot more complicated sadly. As the icon layout in the menu used the span and the svg.